lib/widget/iamport_webview.dart

```diff
@@ -49,9 +51,20 @@ class _IamportWebViewState extends State<IamportWebView> {
   late int _isWebviewLoaded;
   late int _isImpLoaded;
 
+  Future<AndroidDeviceInfo> getAndroidInfo() async {
+    return await DeviceInfoPlugin().androidInfo;
+  }
+
   @override
   void initState() {
     super.initState();
+    Future.delayed(Duration.zero, () async {
+      if (Platform.isAndroid) {
+        var info = await DeviceInfoPlugin().androidInfo;
+        if (info.version.sdkInt == 31)
```

Comment:
```suggestion
if (info.version.sdkInt >= 31) 
```
이 낫지 않을까요? 버전 올라가면 계속 발생할 것 같은 기분이..

Reply:
확인되기 전까지 꼭 수정해야 하는 사항은 아니긴 합니다.

Reply:
삼성폰에 12 오늘 나온거같던데요 ㅋㅋ